bundles/org.openhab.binding.sony/src/main/resources/ESH-INF/config/dial.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0" xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:sony:dialconfig">
+		<parameter name="deviceAddress" type="text" required="true">
+			<context>network-address</context>
+			<label>Dial URL</label>
+			<description>The URL access point for the DIAL service</description>
+		</parameter>
+		<parameter name="deviceMacAddress" type="text">
+			<context>network-address</context>
```

Comment:
Context have a specific meaning used in the user interface. It doesn't look like it's used here in that context. See https://www.openhab.org/docs/developer/bindings/config-xml.html#supported-contexts